src/H5private.h

```diff
@@ -175,6 +160,7 @@
 
 #include <windows.h>
 #include <direct.h> /* For _getcwd() */
+#include <io.h>     /* POSIX I/O */
```

Comment:
io.h is used on Windows for POSIX I/O functionality. It is present on all Windows systems.